assets/js/theme/global.js

```diff
@@ -26,7 +29,9 @@ export default class Global extends PageManager {
         menu();
         mobileMenuToggle();
         privacyCookieNotification();
-        maintenanceMode(this.context.maintenanceMode);
+        if (themeSettings['show-admin-bar']) {
+            adminBar(storeHash, maintenanceModeSettings, JSON.parse(adminBarLanguage), productId, categoryId);
```

Comment:
Instead of defining all these params would it make sense to pass the entire context along with the page type so we can build contextual links for entities based on page type.

Reply:
I thought about that, but I didn't want to inject too much as whatever is injected will show up in HTML; IDs seem non-sensitive. And I'm trying to make this easy to remove before too long so we can remove admin bar from the theme entirely.